### ISSUE TITLE
'Latest' on homepage

### ISSUE THREAD

**Issue by sophie-oms** [sophie-oms/Accessibility-audit#2]
Element has insufficient color contrast of 2.31 (foreground color: #00abc7, background color: #d9f0f5, font size: 27.0pt (36px), font weight: bold). Expected contrast ratio of 3:1
<img width="308" alt="Homepage issue 1" src="https://user-images.githubusercontent.com/77342915/104339419-7910e780-54ef-11eb-8de5-9e99bd33ec85.PNG">
